genet/inputs_handler/matsim_reader.py

```diff
@@ -219,9 +220,15 @@ def write_transitLinesTransitRoute(transitLine, transitRoutes, transportMode):
 
             route = [r_val['link']['refId'] for r_val in transitRoute_val['links']]
 
-            trips = {}
+            trips = {
+                'trip_id': [],
```

Comment:
yeah it's one to one, so at index `x` in each list  there should be a time and vehicle corresponding to that trip_id. It would make sense to make a Trip class and just have a list here but the nestedness of classes is destroying me 😭 `Schedule(Service(Route(Stop)))` is about as much as I can handle. That's partly why I introduced the dataframe setting method. Thinking about it now, it might be good to have something like this when instantiating a `Route`, instead of giving the trips dictionary that is exactly matching those assumptions, you can just give a dataframe with those columns, that would enforce equal length of those lists and would be more user friendly I think

Reply:
The `Trip` class idea was why I was asking this question. I don't think it needs to sit inside a class hierarchy though. You could use a [python dataclass](https://realpython.com/python-data-classes/), maybe, if the class is really just a way to hold data and won't define any logic.

Reply:
ooh, first time I'm seeing this, I like! I'll add a jira card to that effect, thanks!